test/eigen.jl

```diff
@@ -279,4 +279,13 @@ end
     @test λ == [1.0, 8.0]
 end
 
+@testset "SEBA algorithm for real matrices" begin
+    A = [1.0 0.0; 0.0 1.0]
+    B = [cos(pi/3) -sin(pi/3); sin(pi/3) cos(pi/3)]
+    S, R = seba(B*A)
```

Comment:
```suggestion
    S, R = LinearAlgebra.seba(B*A)
```
Since `seba` is not exported, the call needs to be qualified.

Reply:
I added `seba` to be exported now, in `src/LinearAlgebra.jl`, so should work now, thanks for spotting the issue.